src/OpalCompiler-Tests/OCCodeReparatorTest.class.st

```diff
@@ -286,3 +286,31 @@ OCCodeReparatorTest >> testUndeclaredVariable [
 	self assert: method literals first isUndeclaredVariable.
 	self assert: method sourceCode withSeparatorsCompacted equals: 'griffle ^ goo'
 ]
+
+{ #category : #tests }
+OCCodeReparatorTest >> testdefineClass [
+	
+	[ OCCodeReparator new
+		node: (self class>>#testdefineClass) ast;
+		defineClass: 'MyTestClassForDefineClass'
+	]
+		on: ProvideAnswerNotification
```

Comment:
nice. I asked myself how to test UI interactions with the menu. I'm glad you have a solution.